cmd/infracost/run.go

```diff
@@ -60,8 +60,9 @@ func addRunFlags(cmd *cobra.Command) {
 
 	cmd.Flags().Bool("sync-usage-file", false, "Sync usage-file with missing resources, needs usage-file too (experimental)")
 
-	cmd.Flags().Bool("hcl-only", false, "Run infracost without terraform, parsing hcl files at the given path. This option does not need credentials and is much quicker to run.")
-	cmd.Flags().StringSlice("tfvar-files", nil, "Path to .tfvars file, can be used multiple times and evaluated in order of specification")
+	cmd.Flags().Bool("hcl-only", false, "Run infracost without terraform, parsing hcl files at the given path. This option does not need credentials and is much quicker to run. (experimental)")
+	cmd.Flags().StringSlice("var-file", nil, "Can only be used with --hcl-only. Load variable values from the given file, in addition\nto the default files terraform.tfvars and *.auto.tfvars. Use this option more than once to\ninclude more than one variables file. (experimental)")
+	cmd.Flags().StringSlice("var", nil, "Can only be used with --hcl-only. Set a value for one of the input variables in the root module\nof the terraform configuration. Use this option more than once to set more than one variable. (experimental)")
```

Comment:
I'm not reviewing these just now, assuming that finalizing these is part of a different story.

Reply:
👍